packages/swift/build.sh

```diff
@@ -3,7 +3,7 @@ TERMUX_PKG_DESCRIPTION="Swift is a high-performance system programming language"
 TERMUX_PKG_LICENSE="Apache-2.0, NCSA"
 TERMUX_PKG_MAINTAINER="@buttaface"
 TERMUX_PKG_VERSION=5.4.1
-TERMUX_PKG_REVISION=1
+TERMUX_PKG_REVISION=2
```

Comment:
Like I said, don't bump Swift in this pull, as it takes between 3-6 hours to build, so it will just time out this pull on the CI. Once this pull is in, I will bump Swift in a separate commit.

Reply:
This will need a rebase. Or should I push a new commit to undo it (I hope it doesn't build the package)

Reply:
I'm not sure either: try removing the revision bump in a new commit and watch the CI to see if it still builds the Swift package.